js/__fixtures__/contexts/example-v1.json

```diff
@@ -0,0 +1,47 @@
+{
+    "@context": [{
+      "@version": 1.1
+    },"https://www.w3.org/ns/odrl.jsonld", {
+      "ex": "https://example.org/examples#",
+      "schema": "http://schema.org/",
+      "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+  
+      "3rdPartyCorrelation": "ex:3rdPartyCorrelation",
+      "AllVerifiers": "ex:AllVerifiers",
```

Comment:
Is this just a copy of the VC examples context?  I'd prefer not encourage short-form IRIs not that it's not considered best practice, should we change these to long form syntax and then update in the old examples repo as well?

Reply:
yes, this is a copy paste, I like to know exactly what contexts are being used, I would suggest we remove the "example context" when we ship v2 credentials, its terrible.

Reply:
I removed this